samples/sensor/6dof_motion_drdy/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024, TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+# Trigger mode: sample is using interrupt triggering
+choice ICM42670_TRIGGER_MODE
+	default ICM42670_TRIGGER_OWN_THREAD
+endchoice
```

Comment:
This file is for defining new Kconfig symbols, not overriding existing symbols that were defined elsewhere. You should set this in prj.conf instead.

Reply:
Thanks for this clarification. I removed the Kconfig and put this in prj.conf